Artesian/_Query/BidAskQuery.py

```diff
@@ -3,55 +3,131 @@
 from Artesian._Query.Config.ExtractionRangeConfig import ExtractionRangeConfig
 from Artesian._Query.Config.Granularity import Granularity
 from Artesian._Configuration.DefaultPartitionStrategy import DefaultPartitionStrategy
+from Artesian._ClientsExecutor.RequestExecutor import _RequestExecutor
+from Artesian._ClientsExecutor.Client import _Client
+from typing import List
+from __future__ import annotations
 
 import urllib
 class _BidAskQuery(_Query):
     __routePrefix = "ba"
-    def __init__(self, client, requestExecutor, partitionStrategy):
+    def __init__(self, client: _Client, requestExecutor: _RequestExecutor, partitionStrategy: DefaultPartitionStrategy) -> None:
+        """ Inits _BidAskQuery 
+            
+            Args:
+            
+                client
+
+                requestExecutor
+                
+                partitionStrategy. """
         queryParameters = BidAskQueryParameters(None,ExtractionRangeConfig(), None, None, None, None) 
         _Query.__init__(self, client, requestExecutor, queryParameters)
         self.__partition= partitionStrategy
 
-    def forMarketData(self, ids):
+    def forMarketData(self, ids: List[int]) -> _BidAskQuery:
```

Comment:
in _GMEPOfferQuery Returns are documented too. keep Documentation aligned. either document returns everywhere or not.